core/integration/module_elixir_test.go

```diff
@@ -0,0 +1,71 @@
+package core
+
+import (
+	_ "embed"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestModuleElixirInit(t *testing.T) {
+	t.Parallel()
+
+	t.Run("from scratch", func(t *testing.T) {
+		t.Parallel()
+
+		c, ctx := connect(t)
+
+		modGen := daggerCliBase(t, c).
+			With(daggerExec("init", "--name=bare", "--sdk=elixir"))
+
+		out, err := modGen.
+			With(daggerQuery(`{bare{containerEcho(stringArg:"hello"){stdout}}}`)).
+			Stdout(ctx)
+		require.NoError(t, err)
+		require.JSONEq(t, `{"bare":{"containerEcho":{"stdout":"hello\n"}}}`, out)
+	})
+
+	t.Run("camel-cases Dagger module name", func(t *testing.T) {
+		t.Parallel()
+
+		c, ctx := connect(t)
+
+		for _, name := range []string{"My-Module", "MyModule"} {
+			modGen := daggerCliBase(t, c).
+				With(daggerExec("init", "-vv", "--name="+name, "--sdk=elixir"))
```

Comment:
When I remove `-vv` I got 

```
=== NAME  TestModuleElixirInit/camel-cases_Dagger_module_name
    module_elixir_test.go:39:
                Error Trace:    /home/wingyplus/src/github.com/dagger/dagger/core/integration/module_elixir_test.go:39
                Error:          []string{".gitattributes", ".gitignore", "pyproject.toml", "requirements.lock", "sdk", "src"} does not contain "my_module"
                Test:           TestModuleElixirInit/camel-cases_Dagger_module_name
=== NAME  TestModuleElixirInit/with_source
    module_elixir_test.go:65:
                Error Trace:    /home/wingyplus/src/github.com/dagger/dagger/core/integration/module_elixir_test.go:65
                Error:          []string{".gitattributes", ".gitignore", "pyproject.toml", "requirements.lock", "sdk", "src"} does not contain "bare"
                Test:           TestModuleElixirInit/with_source
=== NAME  TestModuleElixirInit/from_scratch
```

Maybe because of I accidentally copy the python sdk at commit https://github.com/dagger/dagger/pull/6967/commits/e3f97af57a7ebfbec4b53420e5a4a2959f25949f. But the issue still occurred even correct the path. After try adding a flag, it works, change the directory name also works! 😢